homeassistant/components/number/const.py

```diff
@@ -132,7 +132,7 @@ class NumberDeviceClass(StrEnum):
     ENERGY = "energy"
     """Energy.
 
-    Unit of measurement: `Wh`, `kWh`, `MWh`, `MJ`, `GJ`
+    Unit of measurement: `Wh`, `kWh`, `MWh`, `MJ`, `GJ`, `GCal`
```

Comment:
This PR is changing our entity model. Before we can review this PR there needs to be approval in a discussion in our architecture repository.

https://github.com/home-assistant/architecture/discussions

https://developers.home-assistant.io/docs/core/entity#changing-the-entity-model

Reply:
Thank, you

So, let's keep track of [architecture repo discussion](https://github.com/home-assistant/architecture/discussions/1018)